src/adfh/ADFH.c

```diff
@@ -2236,6 +2238,13 @@ void ADFH_Database_Open(const char   *name,
     }
   }
 #endif
+  
+  if (0 == strcmp(fmt, "DISKLESS")) {
```

Comment:
See:

CGNS-239 -- Add diskless option for creating a CGNS file

listing the additional items needed for completion of this feature.

Reply:
Oops did not see that coming. Do you want to keep this feature in the branch to add testing or move it to another branch?

Reply:
We should probably move it to another branch.

Reply:
done